### PR TITLE
[POL-1062] Add twilio -> civis connector using parsons library

### PR DESCRIPTION
**Pull request by ted107mk**
Because it's not worth it to add the parsons library to the datascience-python docker image, including instructions to create a container script and use the movementcooperative/parsons docker image.

### CODE REVIEW

File: examples/parsons_civis_twilio_example.py [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+    civis_client.table_import(table, CIVIS_TABLE, existing_table_rows='drop')
+
+
+def read_from_civis():
+    query = f"SELECT * from {CIVIS_TABLE}"
+    table = civis_client.query(query)
+    print(table)
+
+
+twilio_to_civis()
```
Comment (mattyb) [inline examples/parsons_civis_twilio_example.py]: can we add a `validate_params` step to check what's in the environment?

File: examples/parsons_civis_twilio_example.py [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+#         TODO: this would require adding parsons to datascience-python
+
+# Step 2: Set CIVIS_DATABASE and CIVIS_TABLE variables.
+#         CIVIS_TABLE should be in "schema.table" form
+#         See list of databases: platform.civisanalytics.com/spa/remote_hosts
+
+# Step 3: Set environment variables necessary for the clients:
+#         CIVIS_API_KEY: Civis API key
+#         TWILIO_ACCOUNT_SID: Twilio Account SID
+#         TWILIO_AUTH_TOKEN: Twilio Auth Token
```
Comment (mattyb) [inline examples/parsons_civis_twilio_example.py]: Should the auth token be stored in a credential? That's going to look more like an environment variable with `_PASSWORD` at the end?

File: examples/parsons_civis_twilio_example.py [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+
+# Step 1: Open a new Python script in Platform, and paste this entire file
+#         into the script body.
+#         TODO: this would require adding parsons to datascience-python
+
+# Step 2: Set CIVIS_DATABASE and CIVIS_TABLE variables.
+#         CIVIS_TABLE should be in "schema.table" form
+#         See list of databases: platform.civisanalytics.com/spa/remote_hosts
+
+# Step 3: Set environment variables necessary for the clients:
```
Comment (mattyb) [inline examples/parsons_civis_twilio_example.py]: Can we be more explicit how to set up these params for a platform script?

**Review by kmshelley — COMMENTED** [review]
LGTM with Brennan's comments

File: examples/parsons_civis_twilio_example.py [hunk 4]
```diff
@@ -0,0 +1,78 @@
+# This script is an example of using the parsons library to import data from
+# Twilio using the parsons Twilio connector, and then upload the resulting
+# table to Platform using the parsons Civis connector.
+# Below are instructions to be able to customize and run this file in Platform.
+
+# Step 1: Copy this file to a github repository that is connected to Platform.
```
Comment (mattyb) [inline examples/parsons_civis_twilio_example.py]: consistent capitalization of `GitHub`?

File: examples/parsons_civis_twilio_example.py [hunk 5]
```diff
@@ -0,0 +1,78 @@
+# This script is an example of using the parsons library to import data from
+# Twilio using the parsons Twilio connector, and then upload the resulting
+# table to Platform using the parsons Civis connector.
```
Comment (mattyb) [inline examples/parsons_civis_twilio_example.py]: `Parsons` is probably capitalized?

File: examples/parsons_civis_twilio_example.py [hunk 6]
```diff
@@ -0,0 +1,78 @@
...
+#         List of databases: platform.civisanalytics.com/spa/remote_hosts
+#         CIVIS_TABLE should be in "schema.table" form
+
+CIVIS_DATABASE = 99
+CIVIS_TABLE = "twilio_test.account_usage"
+
+# Step 3: Create a platform custom credential with the following fields:
+#         Name: Twilio Account
+#         Credential Type: Custom
+#         Username: Twilio Account SID
```
Comment (mattyb) [inline examples/parsons_civis_twilio_example.py]: put angle brackets around these or something to indicate that they're not literals?

File: examples/parsons_civis_twilio_example.py [hunk 7]
```diff
@@ -0,0 +1,78 @@
...
+#         Update the following fields in the container script:
+#         GITHUB REPOSITORY URL: The github repository from Step 1
+#         GITHUB REPOSITORY REFERENCE: The branch name that contains the copy of this file.
+#         TWILIO ACCOUNT CREDENTIAL: credential from Step 3
+#         Container Scripts docs:
+#         https://civis.zendesk.com/hc/en-us/articles/218200643-Container-Scripts
+
+# Step 5: Run the container script!
+
+# For complete documentation on parsons twilio & civis connectors, see:
```
Comment (mattyb) [inline examples/parsons_civis_twilio_example.py]: capitlize `Civis`

File: examples/parsons_civis_twilio_example.py [hunk 8]
```diff
@@ -0,0 +1,78 @@
...
+#         Name: Twilio Account
+#         Credential Type: Custom
+#         Username: Twilio Account SID
+#         Password: Twilio Auth Token
+#
+#         Docs on creating platform credentials: platform.civisanalytics.com/spa/credentials/new
+#         Note: both twilio variables can be found at www.twilio.com/console
+
+# Step 4: Clone this Platform container script:
+#         platform.civisanalytics.com/spa/#/scripts/containers/100928912
```
Comment (mattyb) [inline examples/parsons_civis_twilio_example.py]: Can we specify the steps instead of recommending the clone?

**Review by mattyb — CHANGES_REQUESTED** [review]
Code's good, just some comments on the docs!

File: examples/parsons_civis_twilio_example.py [hunk 9]
```diff
@@ -1,9 +1,9 @@
-# This script is an example of using the parsons library to import data from
-# Twilio using the parsons Twilio connector, and then upload the resulting
-# table to Platform using the parsons Civis connector.
+# This script is an example of using the Parsons library to import data from
```
Comment (mattyb) [inline examples/parsons_civis_twilio_example.py:1]: sorry, one more nit...can we link to the parsons repo?

**Review by mattyb — CHANGES_REQUESTED** [review]

File: examples/parsons_civis_twilio_example.py (hunk 9 above)
Comment (ted107mk) [reply examples/parsons_civis_twilio_example.py:1]: Done

**Review by kmshelley — APPROVED** [review]
LGTM!

**Review by mattyb — APPROVED** [review]
LGTM!

**Pull request merged**